### PR TITLE
Add February Monthly Update

### PR DESCRIPTION
**Pull request by lauragift21**
A first draft of the monthly update.

Hi, @deivid-rodriguez can you help me provide more context about the priority issue that got fixed and if there are any other updates you'll like to add to February's monthly.

@sonalkr132 can you also help review the RubyGems.org section. 

Thanks!

### CODE REVIEW

File: _posts/2021-03-12-february-rubygems-update.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+In February, [RubyGems](https://github.com/rubygems/rubygems) gained [114 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2021-02-01%7D...master@%7B2021-02-28%7D), contributed by 12 authors. There were 743 additions and 204 deletions across 56 files.
+
+## RubyGems.org News
+
+This month on RubyGems.org, we reduced the ngix rate limit to mitigate the load on Postgres - [#2634]( https://github.com/rubygems/rubygems.org/pull/2634) and evaluated the impact of reducing `random_page_post` in Postgres config.
+
+In addition to that, we made the following improvement and fixes:
+
+- updated elastic search to **v6.8**.
```
Comment (sonalkr132) [inline _posts/2021-03-12-february-rubygems-update.md:24]: Thanks to simi, we significantly improved the delay in our build time by migrating from Travis to Github Actions https://github.com/rubygems/rubygems.org/pull/2625

Comment (lauragift21) [reply _posts/2021-03-12-february-rubygems-update.md:24]: Nice I'll include this.

File: _posts/2021-03-12-february-rubygems-update.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ In addition to that, we made the following improvement and fixes:
 - updated script to block a user from resetting API keys - [#2647](https://github.com/rubygems/rubygems.org/pull/2647)
 - reduced font size of title and subtitle on reverse dep page - [#2637](https://github.com/rubygems/rubygems.org/pull/2637)
 - enabled safe-site lax cookie policy - [#2638](https://github.com/rubygems/rubygems.org/pull/2638) 
-- enabled `return_only_media_type_on_content_type` rails 6.0 default - [#2639](https://github.com/rubygems/rubygems.org/pull/2639) 
+- enabled `return_only_media_type_on_content_type` rails 6.0 default - [#2639](https://github.com/rubygems/rubygems.org/pull/2639)
+- thanks to [simi](https://github.com/simi), we significantly improved the delay in our build time by migrating from Travis to Github Actions. [#2625](https://github.com/rubygems/rubygems.org/pull/2625).
```
Comment (sonalkr132) [inline _posts/2021-03-12-february-rubygems-update.md:30]: sorry, my link was incorrect. it is 2626, not 2625.

File: _posts/2021-03-12-february-rubygems-update.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+This month on RubyGems, we fixed an issue about source priority that received attention due to a popular blog post (link to the blog post?).
+
+As always, we continue to fix bugs, review and merge PR’s and reply to support tickets.
+
+In February, [RubyGems](https://github.com/rubygems/rubygems) gained [114 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2021-02-01%7D...master@%7B2021-02-28%7D), contributed by 12 authors. There were 743 additions and 204 deletions across 56 files.
+
+## RubyGems.org News
+
+This month on RubyGems.org, we reduced the ngix rate limit to mitigate the load on Postgres - [#2634]( https://github.com/rubygems/rubygems.org/pull/2634) and evaluated the impact of reducing `random_page_post` in Postgres config.
```
Comment (sonalkr132) [inline _posts/2021-03-12-february-rubygems-update.md:20]:
```suggestion
This month on RubyGems.org, we reduced the ngix rate limit to mitigate the load on Postgres - [#2634](https://github.com/rubygems/rubygems.org/pull/2634) and evaluated the impact of reducing `random_page_post` in Postgres config.
```

File: _posts/2021-03-12-february-rubygems-update.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+In addition to that, we made the following improvement and fixes:
+
+- updated elastic search to **v6.8**.
```
Comment (sonalkr132) [inline _posts/2021-03-12-february-rubygems-update.md:24]:
```suggestion
- updated Elasticsearch to **v6.8**.
```

**Review by sonalkr132 — APPROVED** [review]

**Comment by deivid-rodriguez** [comment]
@lauragift21 Yes! The link to the blog post is https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610. We have fixed several issues about it this month. In particular, the lockfile has separated rubygems sources now, limiting the issue to fresh installs without a lockfile and allowing for a workaround in the other cases (running `bundle lock` and reviewing the result before installing). The issue is not fully fixed yet, but we continue to work on fully fixing it.

**Comment by lauragift21** [comment]
> @lauragift21 Yes! The link to the blog post is https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610. We have fixed several issues about it this month. In particular, the lockfile has separated rubygems sources now, limiting the issue to fresh installs without a lockfile and allowing for a workaround in the other cases (running `bundle lock` and reviewing the result before installing). The issue is not fully fixed yet, but we continue to work on fully fixing it.

Thanks @deivid-rodriguez.

**Pull request merged**